db/sql/migrations/000027_create_web_vulnerability_instances_table.up.sql

```diff
@@ -1,12 +1,12 @@
 -- Migration: 000027_create_web_vulnerability_instances_table.up.sql
 CREATE TABLE IF NOT EXISTS web_vulnerability_instances (
     id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
```

Comment:
Same here, it will also help debug any cases where we're incorrectly mixing up `id` and `source_vulnerability_id` a little easier.


```suggestion
    id SERIAL PRIMARY KEY,
```

Reply:
Corrected.